### PR TITLE
Import snabblab docs from the wiki into the manual

### LINKED ISSUE #864: update the wiki
Can somebody update the wiki so we know which servers are working and which ones are not. I'm talking about this wiki page:

https://github.com/snabbco/snabb/wiki/Snabb-Lab

Thanks!

### PR DESCRIPTION
**Pull request by domenkozar**
Fixes #864

I've modified it a bit and added minimal `Using the lab` and `Questions` sections.

After this is merged, we should update the wiki to point to the manual.

cc @lukego @eugeneia

### CODE REVIEW

File: src/doc/snabblab.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Please run Snabb processes like this: `sudo lock ./snabb ...`. The `lock` command will automatically wait if somebody else is running a Snabb process on the same machine and that helps us avoid conflicts for access to hardware resources.
+- Tell `luke@snabb.co` your email address(es) to get an invitation to the [Lab Slack](http://snabb.slack.com/).
+- Don't keep precious data on the servers. We might want to reinstall them at short notice.
+
+## Servers
+
+Name | Purpose | SSH | Xeon model | NICs
+---- | ------- | -------- | -------- | ------ | ----
+lugano-1 | General use | lugano-1.snabb.co | E3 1650v3 | 2 x 10G (82599), 4 x 10G (X710), 2 x 40G (XL710)
+lugano-2 | General use | lugano-1.snabb.co | E3 1650v3 | 2 x 10G (82599), 4 x 10G (X710), 2 x 40G (XL710)
```
Comment (plajjan) [inline src/doc/snabblab.md]: isn't fqdn in "SSH" column wrong for this and following rows? I guess name and SSH should match up.

File: src/doc/snabblab.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- Feel at home. These servers are here for you to play with and enjoy.
+- Please run Snabb processes like this: `sudo lock ./snabb ...`. The `lock` command will automatically wait if somebody else is running a Snabb process on the same machine and that helps us avoid conflicts for access to hardware resources.
+- Tell `luke@snabb.co` your email address(es) to get an invitation to the [Lab Slack](http://snabb.slack.com/).
+- Don't keep precious data on the servers. We might want to reinstall them at short notice.
+
+## Servers
+
+Name | Purpose | SSH | Xeon model | NICs
+---- | ------- | -------- | -------- | ------ | ----
```
Comment (plajjan) [inline src/doc/snabblab.md]: Align columns so it's readable in text form as well?

**Comment by plajjan** [comment]
Content wise I think it's good and it addresses #864. I think a link from https://github.com/snabbco/snabb/blame/master/README.md#L91 would be nice as well.

File: src/doc/snabblab.md (hunk 1 above)
Comment (domenkozar) [inline src/doc/snabblab.md]: Good catch!

File: src/doc/snabblab.md (hunk 2 above)
Comment (domenkozar) [inline src/doc/snabblab.md]: Done.

**Comment by domenkozar** [comment]
@plajjan thanks for the review. @lukego do you agree we should mention snabblab in main README.md?

**Comment by domenkozar** [comment]
@eugeneia any idea why it's complaining `Could not find data file .images/Architecture.png`?

**Comment by eugeneia** [comment]
@domenkozar See https://github.com/snabbco/snabb/pull/885

**Comment by eugeneia** [comment]
@domenkozar You should be able to fix the SnabbDoc failure by merging with `master`.

**Comment by domenkozar** [comment]
Yup, it builds now: https://snabbco.github.io/sha1/e5f41835a8b9ef0b8503fc091dd2d9c8824a151e.html#snabblab

**Comment by domenkozar** [comment]
@lukego @eugeneia what's missing here? I'd like to get this into next release so we have something to start with :)

**Pull request merged**